### PR TITLE
feat(button): add style for svg image in button

### PR DESCRIPTION
**Pull request by mtikhinskaya**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/Talend/bootstrap-theme/blob/master/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

Add style for icons on buttons. For example was taken a button from here: http://guidelines.talend.com/document/135460#/1-home

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/theme/_buttons.scss [hunk 1]
```diff
@@ -84,4 +85,11 @@ $btn-background-tint-percent: 10 !default;
 	&-lg {
 		@include button-size($padding-normal, $padding-large-horizontal, $font-size-large, $line-height-large, $btn-border-radius-large);
 	}
+
+	svg {
+		fill: currentColor;
+		float: left;
+		height: 2em;
+		margin: -4px $padding-small -6px 0;
```
Comment (jmfrancois) [inline src/theme/_buttons.scss]: where does values comes from ?
no way to build them from the existing variables ?

Comment (mtikhinskaya) [inline src/theme/_buttons.scss]: svg tag creates frame around picture but padding is 0. This top & border margins just a quick way of fixing it.

File: example/index.html [hunk 2]
```diff
@@ -209,6 +209,44 @@ <h1 id="buttons">Buttons</h1>
               <a href="#" class="btn btn-link disabled">Link</a>
             </p>
 
+            <p id="btn_with_icon" class="bs-component">
```
Comment (jmfrancois) [inline example/index.html]: Could you add btn_with_icon to phantom config so we have a screen shot for it ?

Thanks

File: src/theme/_buttons.scss [hunk 3]
```diff
@@ -84,4 +85,11 @@ $btn-background-tint-percent: 10 !default;
 	&-lg {
 		@include button-size($padding-normal, $padding-large-horizontal, $font-size-large, $line-height-large, $btn-border-radius-large);
 	}
+
+	svg {
+		fill: currentColor;
+		float: left;
```
Comment (frassinier) [inline src/theme/_buttons.scss]: Why `float:left`? What about if I want to add Icon at right?

File: src/theme/_buttons.scss (hunk 1 above)
Comment (frassinier) [inline src/theme/_buttons.scss]: It looks like magic numbers yes!

File: src/theme/_buttons.scss [hunk 4]
```diff
@@ -54,6 +54,7 @@ $btn-background-tint-percent: 10 !default;
 
 	&-primary {
 		@include button-variant($btn-validation-color, $btn-validation-bg, $btn-validation-border);
+		@include btn-colors($btn-validation-bg);
```
Comment (frassinier) [inline src/theme/_buttons.scss]: Why do you need it?

**Pull request merged**